### PR TITLE
add append option to content helper

### PR DESCRIPTION
**Pull request by devsnek**
just stopping with no append is silly. lez fix dis. I added an append option to the helper.
`{{content characters="250" append="..."}}`
`{{content characters="250" append="★"}}`
`{{content characters="250" append="A LONG TIME AGO..."}}`
`{{content words="40" append="limiting with words? ohhhhkay...."}}`
they all work. **well.**

it works with all the builds and tests that it says to run. i tested it with different ascii/hex/random things and it never failed.

### CODE REVIEW

**Comment by GusCaplan** [comment]
to be fair the commit labeled "this time it will work" only failed because of the linter.

**Comment by ErisDS** [comment]
Hi @GusCaplan, thanks for swinging by! 

Can I please point you at a reply I wrote on a recent, similar, feature-adding PR [here](https://github.com/TryGhost/Ghost/pull/6595#issuecomment-203959687) and also at the discussion on [this issue](https://github.com/TryGhost/Ghost/issues/2163) which is all about the feature you've proposed and this [very related issue](https://github.com/TryGhost/Ghost/issues/5060)

TLDR; it's not as simple as it seems but there are some improvements I'd love to see here.

**Comment by GusCaplan** [comment]
@ErisDS I have just added the append option to the excerpt helper, and I plan to add the HTML filter and other features that you mention in [5060](https://github.com/TryGhost/Ghost/issues/5060) later today.

For the future, I think that one of these should be linked to the other because they do the same thing.

**Comment by GusCaplan** [comment]
build 9225 (commit b349156) failed because of apt-get, not my code. [take a look](https://travis-ci.org/TryGhost/Ghost/builds/122859340)

**Comment by GusCaplan** [comment]
i'm moving my content pull and my excerpt pull into seperate PRs so this is now closed.
content PR is [here](https://github.com/TryGhost/Ghost/pull/6727)
